src/test/ui/parser/lex-bad-char-literals-7.rs

```diff
@@ -0,0 +1,14 @@
+// compile-flags: -Z continue-parse-after-error
+fn main() {
+    let _: char = '';
+    //~^ ERROR: empty character literal
+    let _: char = '\u{}';
+    //~^ ERROR: empty unicode escape (must have at least 1 hex digit)
+
+    // Next two are OK, but may befool error recovery
+    let _ = '/';
```

Comment:
This tripped a bug in the error recovery logic, which was caught only by a random run-pass test. Repeating that here to be more robust